test/fixture/generatedTypeScript/interfaces/ICoding.ts

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/prefer-default-export, import/no-cycle, @typescript-eslint/naming-convention, @typescript-eslint/no-empty-interface */
+import { 
+  IElement,
+  IPrimitiveBoolean,
+  IPrimitiveCode,
+  IPrimitiveString,
+  IPrimitiveUri,
+} from "../internal";
+
+export interface ICoding extends IElement {
```

Comment:
Codacy found an issue: [Interface name must not be prefixed with "I".](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5923420)